integrationTests/vm/txsFee/utils/utilsESDT.go

```diff
@@ -308,6 +308,55 @@ func CreateESDTLocalMintTx(nonce uint64, sndAddr, rcvAddr []byte, tokenIdentifie
 	}
 }
 
+// CreateESDTNFTBurnTx -
```

Comment:
we might dissolve this utilsESDT.go along with the utils directory.
This can be a createTestTransactions.go file that can sit in the txFee package, or lower, in vm or integrationTests

Reply:
can be done in another PR